src/main/java/com/salesforce/einsteinbot/sdk/client/BasicChatbotClientImpl.java

```diff
@@ -246,4 +282,23 @@ private Mono<ClientResponse> mapErrorResponse(ClientResponse clientResponse) {
             .error(new ChatbotResponseException(clientResponse.statusCode(), errorDetails,
                 clientResponse.headers())));
   }
+
+  private String getCurrentApiVersion() throws IOException {
+    Properties properties;
+    InputStream is = getClass().getClassLoader()
+        .getResourceAsStream("properties-from-pom.properties");
+    properties = new Properties();
+    properties.load(is);
+    return properties.getProperty("api-spec-yaml-file").substring(1, 6).replace("_", ".");
+  }
```

Comment:
@relango  The maven plugin will copy over the pom properties into this file that is read here to know which api version is being used. I could not find a way to read it as it is hardcoded in the auto-gen code.

Reply:
It's better not  to use the naming convention of file. Is it possible to read api-spec-yaml-file file in maven and use value of `info.version` ?

Reply:
I could look into it, but also, what if we added a new maven property itself like `api-spec-yaml-file`? Something like
```
<api-spec-version>5.0.0</api-spec-version>
```
And then this property can be directly read in the code.